recipes/papillon/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "papillon" %}
+{% set version = "0.0.8" %}
+{% set sha256 = "7505f977ed053b9eff70ba3a6a0787b227f36c419d8d742e3c74d09ebc7bc9c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  skip: True  # [py<35]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python --single-version-externally-managed --record record.txt
+    - setuptools
+    - pandas >=0.17.1
+    - seaborn >=0.8.1
+  run:
+    - python
+    - pandas >=0.17.1
+    - seaborn >=0.8.1
+
+test:
+  imports:
+    - papillon.test
+
+about:
+  home: https://github.com/domenico-somma/Papillon/
+  license: MPL-2.0
+  license_family: OTHER
+  license_file: LICENSE
```

Comment:
It does not look like the `license_file` is included in the source distribution since `LICENSE` is missing from the `MANIFEST.in`. You can manually include it:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and add an issue upstream and add the url as a comment in the `meta.yaml`. Then just include it in future updates to the feedstock when the issue is resolved upstream.

Reply:
I am including it in the next build, thanks.